libs/transactions/src/lib/extension-slot.directive.ts

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
As I understand this would go in foundation-ang. My only concern is that we are trying to reduce (or remove if possible) our core libraries so that we have less things to maintain and customers are more independent from us

Reply:
Yeah, the idea is the `ExtensionComponent` interface and the `ExtensionSlotDirective` abstract class would go somewhere central like in foundation-ang, so RnD teams don't have to duplicate the code in every journey.  This also reduces risk that they'll implement it incorrectly, and enables bugfixes, or Angular API changes like the removal of the use if `ComponentFactoryResolver` (which was deprecated in Ang 13) in one place.